starter_code/models/Order.js

```diff
@@ -0,0 +1,33 @@
+import mongoose from 'mongoose'
+const { Schema, model } = mongoose;
+
+const OrderItemSchema = new Schema({
+    pizza: {
+        type: Schema.Types.ObjectId,
+        ref: 'Pizza'
+    },
+    quantity: {
+        type: Number, min: 1, validate: {
+            validator: Number.isInteger,
+            message: "quantity is not an integer value"
```

Comment:
Nice custom validation!
And preventing negative or zero quantities with min: 1 is actually a nice idea :100: